controllers/operandrequest/operandrequest_controller_test.go

```diff
@@ -160,27 +180,92 @@ var _ = Describe("OperandRegistry controller", func() {
 				return err
 			}, testutil.Timeout, testutil.Interval).Should(Succeed())
 
-			By("Disabling the etcd operator")
-			Eventually(func() bool {
-				requestInstance := &operatorv1alpha1.OperandRequest{}
-				Expect(k8sClient.Get(ctx, requestKey, requestInstance)).Should(Succeed())
-				requestInstance.Spec.Requests[0].Operands = requestInstance.Spec.Requests[0].Operands[1:]
-				Expect(k8sClient.Update(ctx, requestInstance)).Should(Succeed())
+			By("Disabling the etcd operator from first OperandRequest")
+			Eventually(func() error {
+				requestInstance1 := &operatorv1alpha1.OperandRequest{}
+				Expect(k8sClient.Get(ctx, requestKey1, requestInstance1)).Should(Succeed())
+				requestInstance1.Spec.Requests[0].Operands = requestInstance1.Spec.Requests[0].Operands[1:]
+				Expect(k8sClient.Update(ctx, requestInstance1)).Should(Succeed())
 				etcdCluster := &v1beta2.EtcdCluster{}
 				err := k8sClient.Get(context.TODO(), types.NamespacedName{Name: "example", Namespace: operatorNamespaceName}, etcdCluster)
+				return err
+			}, testutil.Timeout, testutil.Interval).Should(Succeed())
+
+			By("Checking etcd operator has not been deleted")
+			Eventually(func() error {
+				etcdCSV := &olmv1alpha1.ClusterServiceVersion{}
+				err := k8sClient.Get(ctx, types.NamespacedName{Name: "etcd-csv.v0.0.1", Namespace: operatorNamespaceName}, etcdCSV)
+				return err
+			}, testutil.Timeout, testutil.Interval).Should(Succeed())
+
+			By("Checking the etcd operator has been deleted")
+			Eventually(func() bool {
+				etcdCSV := &olmv1alpha1.ClusterServiceVersion{}
+				err := k8sClient.Get(ctx, types.NamespacedName{Name: "etcd-csv.v0.0.1", Namespace: operatorNamespaceName}, etcdCSV)
 				return err != nil && errors.IsNotFound(err)
 			}, testutil.Timeout, testutil.Interval).Should(BeTrue())
```

Comment:
I did the same test for `etcd`, this time, `etcd` operator is not deleted.
and the second test is failed because it should return `false` but I set it to be `Should(BeTrue())`

So I would assume it is related to `jenkins` operator itself.

Here is the failed test for the latest test
http://prow.purple-chesterfield.com/log?job=test-operand-deployment-lifecycle-manager-amd64&id=1374222266472075265